Gruntfile.js

```diff
@@ -19,9 +19,48 @@ module.exports = exports = function(grunt) {
                  '    (c) 2013-2015 Mozilla, Apache License 2.0\n' +
                  '*/\n';
 
+    var babelModuleIdProvider = function getModuleId(moduleName) {
+        var files = {
+            'src/localforage': 'localforage',
+            'src/utils/serializer': 'localforageSerializer',
+            'src/drivers/indexeddb': 'asyncStorage',
+            'src/drivers/localstorage': 'localStorageWrapper',
+            'src/drivers/websql': 'webSQLStorage'
+        };
+
+        return files[moduleName] || moduleName.replace('src/', '');
+    };
+
     grunt.initConfig({
+        babel: {
+            options: {
+                loose: 'all',
+                modules: 'umd',
+                moduleIds: true,
+                // sourceMap: true,
+                getModuleId: babelModuleIdProvider,
+                plugins: ['system-import-transformer']
+            },
+            dist: {
+                files: {
+                    'dist/es5src/localforage.js': 'src/localforage.js',
```

Comment:
How about `build/es5src` as a middle ground solution?

On Mon, Jul 13, 2015, 19:15 Matthew Riley MacPherson <
notifications@github.com> wrote:

> In Gruntfile.js
> <https://github.com/mozilla/localForage/pull/390#discussion_r34480366>:
>
> >      grunt.initConfig({
> > +        babel: {
> > +            options: {
> > +                loose: 'all',
> > +                modules: 'umd',
> > +                moduleIds: true,
> > +                // sourceMap: true,
> > +                getModuleId: babelModuleIdProvider,
> > +                plugins: ['system-import-transformer']
> > +            },
> > +            dist: {
> > +                files: {
> > +                    'dist/es5src/localforage.js': 'src/localforage.js',
>
> Works for me, though I’d also be happy if it output to like a .tmp/ folder
> or something. Just to avoid confusion. :-) - tofumatt
> … <#msg-f:1506598354461817953_>
> On Mon, Jul 13, 2015 at 4:36 PM, Thodoris Greasidis <
> notifications@github.com> wrote: > grunt.initConfig({ > + babel: { > +
> options: { > + loose: 'all', > + modules: 'umd', > + moduleIds: true, > +
> // sourceMap: true, > + getModuleId: babelModuleIdProvider, > + plugins:
> ['system-import-transformer'] > + }, > + dist: { > + files: { > +
> 'dist/es5src/localforage.js': 'src/localforage.js', How about just adding
> it to gitignore? Would this be enough? --- Reply to this email directly or
> view it on GitHub:
> https://github.com/mozilla/localForage/pull/390/files#r34475855
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mozilla/localForage/pull/390/files#r34480366>.
>